src/yelp-application.c

```diff
@@ -246,9 +246,13 @@ yelp_application_cmdline (GApplication     *app,
             cur = g_get_current_dir ();
             base = g_file_new_for_path (cur);
             new = g_file_resolve_relative_path (base, (*arguments)[i]);
-            newuri = g_file_get_uri (new);
-            g_free ((*arguments)[i]);
-            (*arguments)[i] = newuri;
+
+            /* Hack to handle --gapplication-service flag. */
```

Comment:
Isn't `--gapplication-service` handled directly by the `GApplication` base class?
This will do sync I/O on the passed-in file during this call, which is probably OK since we're starting up, but it's a bit inelegant.